apps/tup-ui/src/components/auth/LoginComponent/LoginComponent.module.css

```diff
@@ -78,29 +80,31 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-  font-family: var(--global-font-family--serif);
+
+  margin-top: 15px;
 }
 
+
+
+/* FOOTER */
+
 .footer {
   display: flex;
   flex-direction: column;
   align-items: flex-end;
-  margin-top: var(--global-space--small);
-  margin-bottom: var(--global-space--x-small);
+  gap: 12px;
 }
-
-.footer > div {
-  margin-bottom: var(--global-space--small)
+.footer > p {
+  margin-block: 0; /* overwrite browser */
 }
 
+
+/* ERRORS */
+
+/* mimics core-cms django.cms.forms.css .field-errors */
 .error {
-  background-color: var(--global-color-danger--normal);
-  color: var(--global-color-primary--x-light);
-  font-family: var(--global-font-family--sans);
-  font-size: 14px;
-  padding-top: var(--global-space--normal);
-  padding-left: var(--global-space--normal);
-  padding-right: var(--global-space--normal);
-  padding-bottom: var(--global-space--normal);
-  margin-bottom: var(--global-space--normal);
-}
\ No newline at end of file
+  font-size: var(--global-font-size--small);
+
+  margin-top: 1rem;
+  margin-bottom: 0;
+}
```

Comment:
❌ Mimicking CMS did not work out well, because CMS errors are generally un-styled (no design yet) and the location in the markup is different, so we need some of the spacing I deleted.

Reply:
Errors have been styled in recent commits.